app/Http/Controllers/BrandController.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Brand;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Redirect;
+use Illuminate\Support\Facades\Session;
+
+class BrandController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $all_brand_product = Brand::orderby('created_at', 'DESC')->get();
+        return view('brand.all_brand_product')->with(compact('all_brand_product'));
```

Comment:
cách 1 dòng giữa return và các câu lệnh cùng khối lệnh
```suggestion
        $all_brand_product = Brand::orderby('created_at', 'DESC')->get();

        return view('brand.all_brand_product', compact('all_brand_product'));
```